shared/trieutil/sparse_merkle.go

```diff
@@ -194,3 +194,30 @@ func VerifyMerkleBranch(root []byte, item []byte, merkleIndex int, proof [][]byt
 	}
 	return bytes.Equal(root, node[:])
 }
+
+// Copy performs a deep copy of the trie.
+func (m *SparseMerkleTrie) Copy() *SparseMerkleTrie {
+	dstBranches := make([][][]byte, len(m.branches))
+	for i1, srcB1 := range m.branches {
+		dstB1 := make([][]byte, len(srcB1))
```

Comment:
you should use the method here:
https://github.com/prysmaticlabs/prysm/blob/master/shared/bytesutil/bytes.go#L203